target/linux/generic/pending-5.15/730-net-phy-realtek-detect-early-version-of-RTL8221B.patch

```diff
@@ -13,7 +13,15 @@ Signed-off-by: Daniel Golle <daniel@makrotopia.org>
 
 --- a/drivers/net/phy/realtek.c
 +++ b/drivers/net/phy/realtek.c
-@@ -744,6 +744,38 @@ static int rtl8226_match_phy_device(stru
+@@ -79,6 +79,7 @@
+ #define RTLGEN_SPEED_MASK			0x0630
+ 
+ #define RTL_GENERIC_PHYID			0x001cc800
++#define RTL_8221B_VB_CG			0x001cc849
```

Comment:
It's probably better to call that constant
`RTL_8221B_VB_CG_PHYID`
as the other ID constant also got that `_PHYID` suffix. Just to make it acceptable for upstream.

Reply:
Done